src/Aquifer.Jobs/appsettings.example.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Email": {
+        "Marketing": {
+            "Name": {
+                "TemplateId": "sendgrid-id-here"
+            }
+        },
+        "Transactional": {
+            "SendResourceAssignmentNotice": {
+                "TemplateId": "sendgrid-id-here"
+            },
+            "SubscriberNotice": {
```

Comment:
We'll want two sections here, not one (each notification uses a unique template ID):
* `SendProjectStartedNotification`
* `SendResourceCommentCreatedNotification`